docs/authentication.md

```diff
@@ -0,0 +1,48 @@
+## Authentication for Management SDK's
+
+Management SDKs are used to created instances. 
+For those instances we can use OAuth.
+
+All Management SDK's are working with Red Hat Single Sign On (sso.redhat.com) server.
+Authentication to services can be done using Keycloak.js library.
+
+When authenticating please use following keycloak client configuration:
+```
+{
+  "realm": "redhat-external",
+  "auth-server-url": "https://sso.redhat.com/auth/",
+  "ssl-required": "all",
+  "resource": "cloud-services",
+  "public-client": true,
+  "confidential-port": 0
+}
+```
+
+Valid redirect urls: 
+
+`.openshift.com/*, https://prod.foo.redhat.com/*, https://prod.food.redhat.com:1337/*`
```

Comment:
That is applicable to web UIs - will it work for ANY client ID also?

Reply:
That is the client that we provide. Do you mean adding cli client instead?

Reply:
This client will not be useful for golang we should use cli client. Do you mind providing/editing this file with cli client info